### DISCUSSION TITLE
Filter for period of time in ObjectChange

### DISCUSSION THREAD

**Discussion by Lemateus** [netbox-community/netbox#13341] [Help Wanted!]
Hi!
I have this plugin in netbox where I can control inventory. One of the models I use in it is called "Item". Every time an item is updated or created, I can see the information is kept on the ObjectChange model (extras_objectchange table on the DB). What I would like to do is: in the Item's filter, I want to select two dates (begining and end) and get all the items that were updated in that period of time. Is that possible? If so, how can I do it?
Thanks

**Comment by mtinberg** [comment]
Is that not available in the changelog search page? /extras/changelog/  you can filter by model and date along with text search


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: Lemateus ***@***.***>
Sent: Tuesday, August 1, 2023 10:09 PM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Filter for period of time in ObjectChange (Discussion #13341)


Hi!
I have this plugin in netbox where I can control inventory. One of the models I use in it is called "Item". Every time an item is updated or created, I can see the information is kept on the ObjectChange model (extras_objectchange table on the DB). What I would like to do is: in the Item's filter, I want to select two dates (begining and end) and get all the items that were updated in that period of time. Is that possible? If so, how can I do it?
Thanks
